### PR TITLE
Add event and a news item - bwHPC Symposium

### PR DESCRIPTION
**Pull request by sanjaysrikakulam**
Adds an event and a news item regarding the presentation in the 9th bwHPC Symposium. The event item was edited to be posted as earlier.

### CODE REVIEW

File: content/news/2023-11-17-bwhpc-symposium/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: bwHPC Symposium
+date: '2023-11-17'
```
Comment (bgruening) [inline content/news/2023-11-17-bwhpc-symposium/index.md:3]: Date back a week after the event maybe?

Comment (sanjaysrikakulam) [reply content/news/2023-11-17-bwhpc-symposium/index.md]: Done.

Comment (sanjaysrikakulam) [reply content/news/2023-11-17-bwhpc-symposium/index.md]: Wait, does that also mean to predate the directory name? If so I will have to update the directory name,

Comment (bgruening) [reply content/news/2023-11-17-bwhpc-symposium/index.md]: I think so, not really sure how the hub is handling the file name vs. the annotation. Something to try out locally.

**Review by bgruening — APPROVED** [review]

**Pull request merged**